### ISSUE TITLE
fd-multi-input dropdown button submits form

### ISSUE THREAD

**Issue by schaefermichael** [SAP/fundamental-ngx#2083]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
We use a HTML Form with `fd-multi-input`. When clicking the button to open the list of suggestions, the form is being submit. The reason is that `fd-input-group` does not assign a type to the button, making it default to `type="submit"` for most browsers. While there are use-cases where this might make sense, we would need `type="button"` for the `fd-multi-input` dropdown button.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 9, fd-ngx 0.14.0

#### Is there anything else we should know?
Basically the same as https://github.com/SAP/fundamental-ngx/issues/637

**Comment by schaefermichael** [comment]
Same applies for `fd-select` btw
